llvm/lib/Target/AArch64/AArch64InstrInfo.td

```diff
@@ -10378,6 +10378,78 @@ defm : PromoteBinaryv8f16Tov4f32<any_fdiv, FDIVv4f32>;
 defm : PromoteBinaryv8f16Tov4f32<any_fmul, FMULv4f32>;
 defm : PromoteBinaryv8f16Tov4f32<any_fsub, FSUBv4f32>;
 
+//===----------------------------===//
+// 2024 Architecture Extensions:
```

Comment:
Probably worth labelling these as Armv9.6a too, since that's likely to age better.

Reply:
Changed the comment completely to specify, what kind of instruction it includes instead